### ISSUE TITLE
Latest conda seems to have broken nbopen

### ISSUE THREAD

**Issue by bingbong-sempai** [takluyver/nbopen#61]
I recently updated to conda 4.6.4 and nbopen no longer works.
Rolling back the installation fixed nbopen but broke everything else :(

**Comment by ArmandoRl1** [comment]
Can confirm, can't use it.

**Comment by askorikov** [comment]
File manager integration doesn't seem to work in conda 4.7.12 either (running on Windows). On the other hand, `nbopen` command works from Anaconda prompt. I guess, conda has to be activated before running nbopen, which is currently not implemented in the file manager integration, as far as I understand.

**Comment by takluyver** [comment]
I'm not sure why the conda version would affect nbopen at all. But I'm not a Windows user, so I can't really do much to work it out.

**Comment by askorikov** [comment]
In fact, it seems to work if conda is added to the PATH variable (you can choose this option during conda installation).

**Comment by takluyver** [comment]
It should be using a full path to `pythonw.exe` in the registry key, so PATH shouldn't matter. :confused: 

Have a look at the [nbopen.install_win](https://github.com/takluyver/nbopen/blob/master/nbopen/install_win.py) module if you want to dig into it.

**Comment by askorikov** [comment]
Yep, did that. So, on double-click the Python interpreter is launched properly, but without conda environment activation it can't load the required libraries, it seems. Anyway, with the mentioned workaround it works just fine.

**Comment by takluyver** [comment]
That's unfortunate. Normally if you start Python in an environment with a full path, it can access the libraries from that environment. But it sounds like that's no longer the case on Windows.

**Comment by zfoltz** [comment]
Calling `pythonw.exe` from the full path will work but the conda environment will not be activated. This is the same problem if you try to call the anaconda `pythonw.exe` or `python.exe` from command prompt where you will get a warning saying 

> Warning:
> This Python interpreter is in a conda environment, but the environment has
> not been activated.  Libraries may fail to load.  To activate this environment
> please see https://conda.io/activation

The solution to this is to activate the conda environment before starting the python interpreter. I will try to submit a pull request with my working version of  the [nbopen.install_win](https://github.com/takluyver/nbopen/blob/master/nbopen/install_win.py) module later today.

**Comment by Greedquest** [comment]
Anaconda temporarily adds and removes items from the PATH variable when you activate an environment; this is how commands like `nbopen` just work from anaconda prompt but not cmd.exe and why `python file.py` alwas uses the python.exe associated with that conda environment.

The solution I went with was to get conda to run nbopen - see  https://github.com/takluyver/nbopen/issues/69#issuecomment-751688215
